### DISCUSSION TITLE
Queue driver for Dragonfly

### DISCUSSION THREAD

**Discussion by jasonfish568** [laravel/framework#44616] [Ideas]
Just wondering is there currently any plan on intergrating with Dragonfly which is possibly the fastest replacement for redis in the mean time.

**Comment by juse-less** [comment]
I haven't tried it, but had my eyes on Dragonfly for a while.

They claim that they are compatible with both Redis and Memcached.
So you should be able to use existing drivers.
Unless you're looking for the extra features it comes with, obviously.
But thought I'd mention. 🙂

**Reply by jasonfish568** [reply]
Thanks for the info. I was just thinking whether I could use the same driver. I will give it a try.

**Reply by juse-less** [reply]
As I'm, too, interested for the future, let me know how it goes. 🙂
